frontends/web/src/routes/device/bitbox02/wizard.tsx

```diff
@@ -14,211 +14,154 @@
  * limitations under the License.
  */
 
-import { Component } from 'react';
-import { getStatus, getVersion, VersionInfo, verifyAttestation, TStatus } from '../../../api/bitbox02';
+import { useEffect, useState } from 'react';
+import { useLoad, useSync } from '../../../hooks/api';
+import { getStatus, getVersion, verifyAttestation } from '../../../api/bitbox02';
 import { attestationCheckDone, statusChanged } from '../../../api/devicessync';
-import { UnsubscribeList, unsubscribe } from '../../../utils/subscriptions';
 import { route } from '../../../utils/route';
```

Comment:
with this change you could use useNavigation hook instead.

Reply:
Done